Build fails if NPM not installed

I believe the cause of this is adding `npm run test` to the build script. Can we either install npm as well with frontend-maven-plugin, or run the tests with Yarn?

```
[INFO] webpack 5.55.1 compiled with 3 warnings in 23498 ms
[INFO] /bin/sh: npm: command not found
[INFO] error Command failed with exit code 127.
[INFO] info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  06:55 min
[INFO] Finished at: 2022-04-26T14:49:12Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.11.3:yarn (yarn run build) on project cryostat: Failed to run task: 'yarn run build' failed. org.apache.commons.exec.ExecuteException: Process exited with an error: 127 (Exit value: 127) -> [Help 1]
```

--- a/pom.xml
+++ b/pom.xml
@@ -862,9 +862,9 @@
                 <goal>yarn</goal>
               </goals>
               <configuration>
-                  <arguments>
-                      install --frozen-lockfile
-                  </arguments>
+                <arguments>
+                  install --frozen-lockfile
+                </arguments>
               </configuration>
             </execution>
             <execution>
@@ -874,7 +874,18 @@
                 <goal>yarn</goal>
               </goals>
               <configuration>
-                <arguments>run build</arguments>
+                <arguments>run build:notests</arguments>
+              </configuration>
+            </execution>
+            <execution>
+              <id>yarn run test</id>
+              <phase>test</phase>
+              <goals>
+                <goal>yarn</goal>
+              </goals>
+              <configuration>
+                <arguments>run test</arguments>
+                <testFailureIgnore>${skipTests}</testFailureIgnore>
               </configuration>
             </execution>
           </executions>

